canonical-web-and-design/vanilla-framework
# Provide utility to overlap table cell padding

## Done

Provide utility to overlap table cell padding

## QA

- Pull code
- Run `./run serve --watch`
- Open examples/utilities/table-row-padding-overlap/, verify example is correct and classes work as expected

## Details

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2504
Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2463
